frontend/src/Navigation.tsx

```diff
@@ -264,7 +264,7 @@ export default function Navigation({ navOpen, setNavOpen }: NavigationProps) {
               src={UploadIcon}
               onClick={() => setUploadModalState('ACTIVE')}
             ></img>
-            {isDocsListOpen && (
+            {!isDocsListOpen && (
```

Comment:
name of the variable, in that case, should be `isDocsListClosed`, but to prevent negations and rename variables comfier will be to change the ternary [line 258](https://github.com/arc53/DocsGPT/pull/619/files#diff-32e7c0fb6a467d3fa7d164653fb13981bc6ba4a08cba2d8ce1b4ebdac24b2b4dL258)

Reply:
@krabbi I don't think there is need to change the ternary as it has similar functionality to the arrow on navigation.

Please see the ternary at line [184](https://github.com/arc53/DocsGPT/blob/66f2e549ceea8d2af8c7bade657e455d617d290f/frontend/src/Navigation.tsx#L184).

Please correct me if I am wrong.

Reply:
But as @dartpain pointed out the list is open by default so I need to fix that.